pages/recipes/supabase.mdx

```diff
@@ -136,14 +129,14 @@ SNAPLET_SOURCE_DATABASE_URL=<database-url> snaplet ss capture
 
 ```ts snaplet.config.ts
 generate: {
-    plan({ snaplet, pipe }) {
-      return pipe([
-        // 10 users
-        snaplet.users({ count: 10 }),
-        // 20 todos, with one of the 10 users assigned to each todo
-        snaplet.todos({ count: 20 }, { autoConnect: true })
-      ]);
-    },
+  async run(snaplet) {
+    await snaplet.$pipe([
+      // 10 users
+      snaplet.users((x) => x(5)),
+      // 20 todos, with one of the 10 users assigned to each todo
+      snaplet.todos((x) => x(10), { autoConnect: true }),
```

Comment:
here you changed the numbers but not the comments.

Reply:
Thanks for catching, will update, the comments are right.